<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(87, 142, 184);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(110, 181, 235);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>120</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);
border-radius: 12px;
padding-left: 15px;
}

QLineEdit:focus{
	border: 2px solid #32536B;
}</string>
          </property>
          <property name="placeholderText">
           <string>Nome Do Produto</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>180</y>
            <width>221</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);
border-radius: 12px;
padding-left: 15px;
}

QLineEdit:focus{
	border: 2px solid #32536B;
}</string>
          </property>
          <property name="placeholderText">
           <string>Pre√ßo</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>240</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);
border-radius: 12px;
padding-left:15px;
}

QLineEdit:focus{
	border: 2px solid #32536B;
}</string>
          </property>
          <property name="placeholderText">
           <string>Quantidade</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>320</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border: none;
	background-color:#B7D6EE;
	border-radius: 12px;
	
}
QPushButton:hover{
	background-color:#52606B;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>40</y>
            <width>342</width>
            <height>32</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>CADASTRAR PRODUTO</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(110, 184, 236);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_5">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
